addons/website_forum/models/forum_forum.py

```diff
@@ -146,6 +149,13 @@ def _compute_has_pending_post(self):
         pending_forums.has_pending_post = True
         (self - pending_forums).has_pending_post = False
 
+    can_moderate = fields.Boolean(string="Is a moderator", compute="_compute_can_moderate")
```

Comment:
Don't mix methods and fields, should be defined in the fields list of the model.